account_invoice_date/README.rst

```diff
@@ -0,0 +1,6 @@
+====================
```

Comment:
This file should not be included, it'll be autogenerated later

Reply:
Hi, @dalonsod when I remove the " README.rst" file and then I've run the "pre-commit run -a" command faced an issue in the " README.rst" file. Please find the attached image.
![README_issue](https://user-images.githubusercontent.com/106082361/178907865-794a4ec8-bb9f-4fa9-85a4-749ce09f691e.png)